components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -798,17 +989,25 @@ IJsonSerializer jsonSerializer
         /// <inheritdoc/>
         public async Task<List<OpcNodeModel>> GetConfiguredNodesOnEndpointAsync(
             PublishedNodesEntryModel request,
-            CancellationToken ct = default) {
-
+            CancellationToken ct = default
+        ) {
```

Comment:
is this necessary? I think we should agree on a coding style and keep look and feel consistent. When in Rome ...

Reply:
Nop, not necessary. Will revert this change.

Reply:
Reverted.